guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>remove the seq_len=12, since this was a particular case, and could be generalised as seq_len=n</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>